src/Dropzone/components/DropzoneButtons/DropzoneButtons.tsx

```diff
@@ -37,28 +37,28 @@ const DropzoneButtons: React.FC<DropzoneButtonsProps> = (
     cleanButton
       ? {
           ...cleanButton,
-          label: "Clean",
+          label: cleanButton.label || "Clean",
```

Comment:
The old code would override any custom label. The changes only override falsy labels.